astrokat/utility.py

```diff
@@ -41,7 +41,10 @@ def read_yaml(filename):
         if instrument is not None:
             if "integration_time" in instrument.keys():
                 integration_time = float(instrument["integration_time"])
-                instrument["dump_rate"] = 1.0 / integration_time
+                if integration_time == 1.33:
+                    instrument["dump_rate"] = 0.75
```

Comment:
These magic values are not ideal - likely to require future maintenance, and it can easily be avoided.  We can stick with the old code here, and just fix the comparison in `Telescope.subarray_setup`, which is the only place it seems to be used.

Reply:
done